package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
-    "coverage": "nyc report --reporter=text",
+    "test": "npm run undo:test && npm run migrate:test && npm run seed:test && cross-env NODE_ENV=test nyc mocha --require @babel/register ./src/__tests__/*.js --timeout 20000",
```

Comment:
why this, we will have to conclude on the test script we will all use

Reply:
This is to reset the database before running the .